src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  max-width: calcCalendarWidth($day-width-height, $gap);
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    @include square($day-width-height);
+    background-color: $color-day-bg;
+    border: 1px solid $color-day-border;
+    display: flex;
+    margin-right: $gap;
+    margin-bottom: $gap;
+    cursor: pointer;
+    font-family: "Arial", sans-serif;
+    font-size: 30px;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: #fcbfcb;
+      transform: translateY(-20px);
+    }
+
+    @for $day-number from 1 through 31 {
+      &:nth-child(7n) {
+        margin-right: 0;
+      }
```

Comment:
does it make sense? as i see (devtools) this rule is always covered by another rules. mb i'm not right but it seems so

Reply:
true, unnecessary styles

Reply:
> does it make sense? as i see (devtools) this rule is always covered by another rules. mb i'm not right but it seems so

true, unnecessary styles